glue_ginga/plugins/Glue.py

```diff
@@ -134,18 +219,23 @@ def put_data_cb(self):
         kwargs = {name: data_np}
 
         try:
+            # TODO: Pass in WCS for image.
+            #if isinstance(image, AstroImage.AstroImage):
+            #    ???.coords.wcs = image.wcs???
```

Comment:
Thanks for the suggestions. Let me see if I can wrap this up today. @ejeschke , let's postpone merging with `client.py` for another PR. I don't have time for that today.